llvm/lib/Target/AIE/AIELegalizerHelper.h

```diff
@@ -0,0 +1,70 @@
+//===- AIELegalizerHelper ---------------------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2023-2024 Advanced Micro Devices, Inc. or its affiliates
```

Comment:
nit: AIELegalizerHelper.cpp states just `2024`

Reply:
2023-2024 is in this correct, I think, you are moving AMD code from 2023 into this file, so it should keep the same date range.